benchmarks/babelstream/src/babelStreamMainTest.cpp

```diff
@@ -267,210 +296,328 @@ void testKernels()
         bufAccInputBPtr,
         bufAccOutputCPtr);
 
-    // Vector of average run-times of babelstream kernels
-    std::vector<double> avgExecTimesOfKernels;
-    std::vector<double> minExecTimesOfKernels;
-    std::vector<double> maxExecTimesOfKernels;
-    std::vector<std::string> kernelLabels;
-    // Vector for collecting successive run-times of a single kernel in benchmark macro
-    std::vector<double> times;
+    auto const workDivNStream = alpaka::getValidWorkDiv(
+        kernelCfg,
+        devAcc,
+        NstreamKernel(),
+        bufAccInputAPtr,
+        bufAccInputBPtr,
+        bufAccOutputCPtr);
```

Comment:
Since alpaka can run on different hardware it is fine to calculate the workdiv and not use hard-coded values. 
Hardcoding per hardware/ backend is possible too but not nice.

Reply:
Okay. Resolved.